ISHPermissionKit/Requests/ISHPermissionRequestEventStore.m

```diff
@@ -55,7 +55,8 @@ - (void)requestUserPermissionWithCompletionBlock:(ISHPermissionRequestCompletion
     
     [self.eventStore requestAccessToEntityType:[self entityType] completion:^(BOOL granted, NSError *error) {
         dispatch_async(dispatch_get_main_queue(), ^{
-            completion(self, granted ? ISHPermissionStateAuthorized : ISHPermissionStateDenied, error);
+            NSError *externalError = (error.code == EKErrorEventStoreNotAuthorized) ? nil : error;
```

Comment:
`NSError *externalError = (error && (error.code != EKErrorEventStoreNotAuthorized)) ? error : nil;` seems safer to me. Else this could behave unexpectedly if `EKErrorEventStoreNotAuthorized` was `0` I believe.